content/_code-samples/non-fungible-token/py/list-nft-pages-and-offers.py

```diff
@@ -0,0 +1,54 @@
+from xrpl.models.requests.account_nfts import AccountNFTs
```

Comment:
I'd recommend renaming this to `list-nft-pages-and-buy-offers.py` since it only checks for buy offers, not sell offers. I was confused when I ran it after creating a sell-offer for my NFT and then it said "...no buy offers..." (Took me a second to realize it was talking about buy offers, instead of saying "no offers" in general)